src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -236,6 +254,33 @@ private void resendClaimsToRPA(List<Claim> claims, String authorisation) {
         }
     }
 
+    private void resendStaffNotificationClaimantResponse(Claim claim) {
+        ClaimantResponse claimantResponse = claim.getClaimantResponse()
+            .orElseThrow(IllegalStateException::new);
+        Response response = claim.getResponse().orElseThrow(IllegalArgumentException::new);
```

Comment:
Is there really a difference between a missing claimant response and a missing response? I think they should both be `IllegalArgumentException`s